application/clicommands/WebCommand.php

```diff
@@ -21,8 +40,14 @@ public function serveAction()
         }
 
         $fork = $this->params->get('daemonize');
+        $listen = $this->params->get('listen');
+        $port = $this->params->get('port');
         $documentRoot = $this->params->shift();
-        $socket  = $this->params->shift();
+        if ($listen === null) {
+            $socket = $port === null ? $this->params->shift() : '0.0.0.0:' . $port;
+        } else {
+            $socket = $listen;
+        }
 
         // TODO: Sanity check!!
```

Comment:
Hi Alex,

Please remove the TODO comment as PHP already fails w/ "Invalid address: ...", if the address is invalid.

Best regards,
Eric